daemon/daemon_config.c

```diff
@@ -89,6 +89,8 @@ struct GameModeConfig {
 
 		char apply_gpu_optimisations[CONFIG_VALUE_MAX];
 		long gpu_device;
+		char vsync_mode[CONFIG_VALUE_MAX];
+		char hybrid_gpu_mode[CONFIG_VALUE_MAX];
```

Comment:
The same ordering issue config example exists here - it'd be best to move them to distinguish them. Perhaps in a whole new config section `[gamemoderun] `

Reply:
Agree with that.